content/concepts/nft.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Data NFTs
+description: Ocean Protocol uses ERC721 NFTs to represent the intellectual property (IP) of published datasets or algorithms, and to further sub-license access to it with user roles or ERC20 data tokens.
+---
+
+A non-fungible token stored on the blockchain represents a unique asset. NFTs can represent images, videos, digital art, or any piece of information. NFTs can be traded and allow transfer of ownership and copyright. [EIP-721](https://eips.ethereum.org/EIPS/eip-721) defines an interface for handling NFTs on Ethereum blockchain. The creator of the NFT can deploy a new contract on Ethereum or any Blockchain supporting NFT related interface and also, transfer the ownership through transfer transactions.
+
+## Terminology
+
+- **Base IP** means the artifact being copyrighted. Represented by the {ERC721 address, tokenId} from the publish transactions.
```

Comment:
asset, dataset, algorithm, artifact. We use a lot of different names for the same thing so in this case let's just stick to "asset", and NOT introduce yet another term with "artifact".

Also "Base IP" feels like a confusing term if we never write out "IP". Is it "Intellectual Property"?

Reply:
Agreed. Click on the "Files changed" tab and you'll see specific recommendations that I'm making on exactly this.